coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -279,8 +228,9 @@ export default {
     },
 
     directorsChange () {
-      if (this.directorsChangeCount > 0) return true
-      else return false
+      // One or more actions taken on directors (add, cease) require a single fee, so check how many directors in the
+      // list are marked as requiring a fee.
+      return this.directors.filter(director => director.isFeeApplied).length > 0
```

Comment:
I'm confused: a single fee for any number of dir changes, or 1 fee per changed director? If the latter then it will discourage ceasing and re-adding a director (eg, to correct their name or address).
Thoughts?